src/__experimental__/components/checkbox/checkbox.component.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import tagComponent from '../../../utils/helpers/tags';
+import { validProps } from '../../../utils/ether';
+import FormField from '../form-field';
+import CheckboxStyle from './checkbox.style';
+
+const Checkbox = (props) => {
+  const formFieldProps = () => {
+    const { ...validFormFieldProps } = validProps(
+      { propTypes: Checkbox.propTypes, safeProps: ['fieldHelpInline'], props }
+    );
+
+    validFormFieldProps.reverse = !props.reverse;
+
+    return validFormFieldProps;
+  };
+
+  const inputProps = () => {
+    const { ...validInputProps } = validProps(
+      { propTypes: Checkbox.propTypes, safeProps: ['disabled', 'onChange'], props }
+    );
+
+    validInputProps.type = 'checkbox';
+    delete validInputProps.children;
```

Comment:
You could use destructuring to omit these properties rather than removing them declaratively. 

```js
const { children, fieldHelp, labelHelp, ...rest }  = validProps(/* */);
const validInputProps = { ...rest, type: 'checkbox' };
```

Reply:
👍